### PR TITLE
Release v0.5.0

### LINKED ISSUE #100: Create a new release to distribute SRC14
https://github.com/FuelLabs/sway/pull/6069 -- tooling support for proxy contracts require to pin this branch to a version that includes SRC14. So we require a new release.

### PR DESCRIPTION
**Pull request by bitzoic**
## Type of change

- Release

## Changes

The following changes have been made:

- Merge master into release

## Notes

- No breaking changes

## Related Issues

Closes #100

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,8 +9,8 @@
     <a href="https://github.com/FuelLabs/sway-standards/actions/workflows/ci.yml" alt="CI">
         <img src="https://github.com/FuelLabs/sway-standards/actions/workflows/ci.yml/badge.svg" />
     </a>
-    <a href="https://crates.io/crates/forc/0.49.1" alt="forc">
-        <img src="https://img.shields.io/badge/forc-v0.49.1-orange" />
+    <a href="https://crates.io/crates/forc/0.59.0" alt="forc">
```
Comment (SwayStar123) [inline README.md:12]:
```suggestion
    <a href="https://crates.io/crates/forc/0.60.0" alt="forc">
```

File: README.md [hunk 2]
```diff
@@ -9,8 +9,8 @@
...
+        <img src="https://img.shields.io/badge/forc-v0.59.0-orange" />
```
Comment (SwayStar123) [inline README.md:13]:
```suggestion
        <img src="https://img.shields.io/badge/forc-v0.60.0-orange" />
```

File: README.md [hunk 3]
```diff
@@ -147,10 +147,10 @@ Example of the SRC-12 implementation where contract deployments contain configur
 
 ##### [Without Configurables](./examples/src12-contract-factory/without_configurables/src/without_configurables.sw)
 
-Example of the SRC-12 implementation where all contract deployments are identitcal and thus have the same bytecode and root.
+Example of the SRC-12 implementation where all contract deployments are identical and thus have the same bytecode and root.
 
 > **Note**
-> All standards currently use `forc v0.56.0`.
+> All standards currently use `forc v0.59.0`.
```
Comment (SwayStar123) [inline README.md:153]:
```suggestion
> All standards currently use `forc v0.60.0`.
```

**Review by SwayStar123 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bitzoic) [reply README.md:12]: Resolved by https://github.com/FuelLabs/sway-standards/pull/102

File: README.md (hunk 3 above)
Comment (bitzoic) [reply README.md:153]: Resolved by https://github.com/FuelLabs/sway-standards/pull/102

**Review by SwayStar123 — APPROVED** [review]

**Review by K1-R1 — APPROVED** [review]

**Pull request merged**